index.html

```diff
@@ -486,9 +486,9 @@
 conveying objections to [=processing=] based on legitimate interest or the withdrawal of
 [=consent=] to specific [=purposes=].) It should be noted that, since a [=global opt-out=]
 signal is reaffirmed automatically with every interaction, it will take precedence
-in terms of specificity over any manner of blanket [=consent=] that a site may obtain,
-unless that [=consent=] is directly attached to an interaction (eg. terms specified on a
-form upon submission).
+in terms of specificity over any general assertion of [=consent=] by a site,
+and only superseded by specific [=consent=] obtained through a deliberate action taken by
+the user with the intent of overriding their global opt-out.
```

Comment:
The intent isn't to make a legal claim here but rather to explain the conceptual model that we think applies in terms of what takes precedence in case of conflict.